docs/topics/gradle.md

```diff
@@ -662,7 +662,7 @@ Here is a complete list of options for Gradle tasks:
 |------|-------------|-----------------|--------------|
 | `allWarningsAsErrors` | Report an error if there are any warnings |  | false |
 | `suppressWarnings` | Don't generate warnings |  | false |
-| `verbose` | Enable verbose logging output |  | false |
+| `verbose` | Enable verbose logging output; works only when the [Gradle debug log level enabled](https://docs.gradle.org/current/userguide/logging.html) |  | false |
```

Comment:
Why did you decide not to write a separate sentence?

Reply:
Discussed in DM.